variables.tf

```diff
@@ -42,7 +42,7 @@ variable "database_name" {
 variable "deletion_protection" {
   description = "If the DB instance should have deletion protection enabled. The database can't be deleted when this value is set to true."
   type        = bool
-  default     = false
+  default     = true
```

Comment:
Er det nødvendig å endre hva default-verdien er? Kan muligens være enig i at den burde vært `true` fra starten av, men dette vil introdusere en endring for alle som ikke har eksplisitt satt en verdi.

Reply:
Vil denne endringen ha noen annen konsekvens enn at de som oppgraderer avhengighet til denne modulen må eksplisitt slå av deletion protection? Isåfall tenker jeg det bare er en bra ting.